pkg/metaserver/agent/pod/kubelet.go

```diff
@@ -35,18 +35,26 @@ type KubeletPodFetcher interface {
 // kubeletPodFetcherImpl use kubelet 10255 pods interface to get pod directly without cache.
 type kubeletPodFetcherImpl struct{}
 
-// GetPodList get pods from kubelet 10255/pods api, and the returned slice does not
-// contain pods that don't pass `podFilter`
-func (k *kubeletPodFetcherImpl) GetPodList(_ context.Context, podFilter func(*v1.Pod) bool) ([]*v1.Pod, error) {
+func getPodsByKubeletAPI() (v1.PodList, error) {
```

Comment:
why we need to extract this function out?

Reply:
No need any more, removed already.